tests/system-tests/rdscore/internal/rdscorecommon/sriov-pod-level-bond.go

```diff
@@ -0,0 +1,1197 @@
+}
+
+func verifyConnectivity() {
+	verifyPodLevelBondWorkloads(
+		RDSCoreConfig.PodLevelBondDeploymentOneName,
+		RDSCoreConfig.PodLevelBondNamespace,
+		RDSCoreConfig.PodLevelBondDeploymentTwoName,
+		RDSCoreConfig.PodLevelBondNamespace,
+		RDSCoreConfig.PodLevelBondDeploymentOneIPv4,
+		RDSCoreConfig.PodLevelBondDeploymentOneIPv6,
+		RDSCoreConfig.PodLevelBondDeploymentTwoIPv4,
+		RDSCoreConfig.PodLevelBondDeploymentTwoIPv6)
+}
+
+// VerifyPodLevelBondWorkloadsOnSameNodeSamePF verifies TCP traffic works on the same node and different PFs.
+func VerifyPodLevelBondWorkloadsOnSameNodeSamePF() {
+	prepareSecondPodLevelBondDeployment(true, true)
+
+	verifyConnectivity()
+}
+
+// VerifyPodLevelBondWorkloadsOnSameNodeDifferentPFs verifies TCP traffic works on the same node and different PFs.
+func VerifyPodLevelBondWorkloadsOnSameNodeDifferentPFs() {
+	prepareSecondPodLevelBondDeployment(true, false)
+
+	verifyConnectivity()
+}
+
+// VerifyPodLevelBondWorkloadsOnDifferentNodesSamePF verifies TCP traffic works on the different nodes and same PF.
+func VerifyPodLevelBondWorkloadsOnDifferentNodesSamePF() {
+	prepareSecondPodLevelBondDeployment(false, true)
+
+	verifyConnectivity()
+}
+
+// VerifyPodLevelBondWorkloadsOnDifferentNodesDifferentPFs verifies TCP traffic works on the
+// different nodes and different PFs.
+func VerifyPodLevelBondWorkloadsOnDifferentNodesDifferentPFs() {
+	prepareSecondPodLevelBondDeployment(false, false)
+
+	verifyConnectivity()
+}
+
+// VerifyPodLevelBondWorkloadsAfterVFFailOver verifies TCP traffic after bond active interface failure
+// (fail-over procedure).
+//
+//nolint:funlen
+func VerifyPodLevelBondWorkloadsAfterVFFailOver() {
+	prepareSecondPodLevelBondDeployment(false, true)
+
+	verifyConnectivity()
+
+	By("Retrieve client pod-level bond pod object")
+
+	clientPodObj, err := getPodObjectByNamePattern(
+		APIClient,
+		RDSCoreConfig.PodLevelBondDeploymentOneName,
+		RDSCoreConfig.PodLevelBondNamespace)
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Failed to retrieve client pod-level bond %s object from namespace %s: %v",
+			RDSCoreConfig.PodLevelBondDeploymentOneName, RDSCoreConfig.PodLevelBondNamespace, err))
+
+	By("Retrieve server pod-level bond pod object")
+
+	serverPodObj, err := getPodObjectByNamePattern(
+		APIClient,
+		RDSCoreConfig.PodLevelBondDeploymentTwoName,
+		RDSCoreConfig.PodLevelBondNamespace)
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Failed to retrieve server pod-level bond %s object from namespace %s: %v",
+			RDSCoreConfig.PodLevelBondDeploymentOneName, RDSCoreConfig.PodLevelBondNamespace, err))
+
+	activeInf, err := getBondActiveInterface(serverPodObj)
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Failed to retrieve bond active interface for the pod deployment %s in namespace %s: %v",
+			serverPodObj.Definition.Name, serverPodObj.Definition.Namespace, err))
+
+	glog.V(100).Infof("DEBUG LOG: active interface found - %s", activeInf)
```

Comment:
```suggestion
	glog.V(100).Infof("Active interface found - %s", activeInf)
```

Reply:
ups, sorry, it was debug stage left-overs. I remove this like we have this info printed-out inside  getBondActiveInterface method